src/Application/Commands/TakeRegularGivingDonations.php

```diff
@@ -101,6 +102,19 @@ private function createNewDonationsAccordingToRegularGivingMandates(\DateTimeImm
         }
     }
 
+    private function setPaymentIntentWhenReachedPaymentDate(
+        \DateTimeImmutable $now,
+        SymfonyStyle $io
+    ): void {
+        $donations = $this->donationRepository->findDonationsToSetPaymentIntent($now, limit:20);
```

Comment:
Is there a particular reason to match `findPreAuthorizedDonationsReadyToConfirm()`'s limit? If both are going to use the same limit long-term it seems like it should be a class constant.